getting-started/setup-building.rst

```diff
@@ -414,27 +414,31 @@ OpenSSL which means that you will not be able to build the ``_ssl`` extension.
 One solution is to install these libraries from a third-party package
 manager, like Homebrew_ or MacPorts_, and then add the appropriate paths
 for the header and library files to your ``configure`` command.  For example,
+with **Homebrew**,
 
-with **Homebrew**::
+For Python 3.10 and newer::
 
-    $ brew install pkg-config openssl@1.1 xz gdbm tcl-tk
+    $ brew install pkg-config openssl xz gdbm tcl-tk
 
-For Python 3.10 and newer::
+and ``configure``::
 
     $ CFLAGS="-I$(brew --prefix gdbm)/include -I$(brew --prefix xz)/include" \
       LDFLAGS="-L$(brew --prefix gdbm)/lib -I$(brew --prefix xz)/lib" \
-      PKG_CONFIG_PATH="$(brew --prefix tcl-tk)/lib/pkgconfig" \
       ./configure --with-pydebug \
-                  --with-openssl=$(brew --prefix openssl)
+                  --with-openssl="$(brew --prefix openssl)"
 
 
 For Python versions 3.9 through 3.7::
 
-    $ CFLAGS="-I$(brew --prefix gdbm)/include -I$(brew --prefix xz)/include" \
+    $ brew install pkg-config openssl@1.1 xz gdbm tcl-tk
+
+and ``configure``::
+
+    $ export PKG_CONFIG_PATH="$(brew --prefix tcl-tk)/lib/pkgconfig"; \
+      CFLAGS="-I$(brew --prefix gdbm)/include -I$(brew --prefix xz)/include" \
       LDFLAGS="-L$(brew --prefix gdbm)/lib -L$(brew --prefix xz)/lib" \
-      PKG_CONFIG_PATH="$(brew --prefix tcl-tk)/lib/pkgconfig" \
       ./configure --with-pydebug \
-              --with-openssl=$(brew --prefix openssl@1.1) \
+              --with-openssl="$(brew --prefix openssl@1.1)" \
               --with-tcltk-libs="$(pkg-config --libs tcl tk)" \
               --with-tcltk-includes="$(pkg-config --cflags tcl tk)"
 
```

Comment:
Can't reach it with a suggestion, but the current syntax isn't explicit that "and make" applies to both branches, rather than just the latter as the structure would imply (particularly prior to my suggestions). I would also make similar clarifications with the "MacPorts" section, though maybe getting a little out of scope here.

Reply:
Implemented in followup review